### PR TITLE
Admin portal UI

### PR DESCRIPTION
**Pull request by ttishkin**

### CODE REVIEW

File: src/CSI.IBTA.Shared/Types/TaskResult.cs [hunk 1]
```diff
@@ -0,0 +1,9 @@
+﻿
+namespace CSI.IBTA.Shared.Types
```
Comment (radislavkireev) [inline src/CSI.IBTA.Shared/Types/TaskResult.cs:2]: not using and could be removed

Comment (VidasBui) [reply src/CSI.IBTA.Shared/Types/TaskResult.cs:2]: it is being used. I suppose that you were talking about FileUtils

Comment (radislavkireev) [reply src/CSI.IBTA.Shared/Types/TaskResult.cs:2]: Yeah, I see the call of CreateEmployer. 
The classes GenericHttpResponse, GenericInternalResponse, TaskResult, GenericResponse are confusing and doing the same task actually. 

Can you leave one of them and drop the rest? Does not matter which class but lets make it universal and do not maintain that zoo. 

Current PR could be merged, I am not rejecting it. 
After the merge, lets do one more PR, that solves the responses issue. Do not wait my response on both PRs, feel free to merge. 

cc @mykytatishkin @VidasBui

File: src/CSI.IBTA.Administrator/Views/Shared/_Employer.cshtml [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+                <a asp-action="Index"
+                   asp-route-sortOrder="@ViewData["CurrentSort"]"
+                   asp-route-pageNumber="@(Model.PageIndex - 1)"
+                   asp-route-currentFilter="@ViewData["CurrentFilter"]"
+                   class="btn btn-default @prevDisabled">
+                    Previous
+                </a>
+                @{
+                    var lastPage = Math.Min(Model.PageIndex + 7, Model.TotalPages);
+                    var firstPage = Math.Max(1, lastPage - 9);
```
Comment (radislavkireev) [inline src/CSI.IBTA.Administrator/Views/Shared/_Employer.cshtml]: What kind of magic is here +7 -9 ?

**Review by VidasBui — APPROVED** [review]

**Review by ugis90 — APPROVED** [review]

**Pull request merged**